packages/web/src/components/data-entry/ContextualMenu.tsx

```diff
@@ -55,17 +55,15 @@ const MenuForm = (props: MenuFormProps) => {
         <ModalTitle title={label} icon={icon} />
       </ModalHeader>
       <ModalContent>
-        <Form>
-          {menuFields}
-          <ModalFooter>
-            <Button
-              type={ButtonType.PRIMARY}
-              text={messages.save}
-              buttonType='submit'
-            />
-          </ModalFooter>
-        </Form>
+        <Form>{menuFields}</Form>
       </ModalContent>
+      <ModalFooter>
+        <Button
+          type={ButtonType.PRIMARY}
+          text={messages.save}
+          buttonType='submit'
```

Comment:
we're no longer inside the form, so need to do the onClick more explicitly

Reply:
yes